### PR TITLE
feat(graphql): implement create immnDosage

### PR DESCRIPTION
**Pull request by progsmile**
#4185

### CODE REVIEW

File: apps/core/lib/core/medications/medications.ex [hunk 1]
```diff
@@ -396,14 +396,14 @@ defmodule Core.Medications do
 
   # Create
 
-  def create_innm_dosage(attrs, headers), do: create_medication_entity(INNMDosage, attrs, headers)
+  def create_innm_dosage(attrs, actor_id), do: create_medication_entity(INNMDosage, attrs, actor_id)
```
Comment (PavelVesnin) [inline apps/core/lib/core/medications/medications.ex]: for consistency, actor_id should be a first argument

Comment (progsmile) [reply apps/core/lib/core/medications/medications.ex]: suppose it's unlogical to put actor_id first

Comment (progsmile) [reply apps/core/lib/core/medications/medications.ex]: in elixir data always first, than some config stuff

Comment (PavelVesnin) [reply apps/core/lib/core/medications/medications.ex]: than it should be everywhere. But now actor_id is first argument

Comment (progsmile) [reply apps/core/lib/core/medications/medications.ex]: as you may notice there are more cases with `actor_id` last in `Medications`, so no consistency there

Comment (PavelVesnin) [reply apps/core/lib/core/medications/medications.ex]: https://github.com/edenlabllc/ehealth.api/blob/develop/apps/core/lib/core/medical_programs/medical_programs.ex#L47
https://github.com/edenlabllc/ehealth.api/blob/develop/apps/core/lib/core/medications/medications.ex#L551
and others

**Pull request merged**